models/placement.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+const sequelize = require('../config/sequelize')
+const {reduce, replace} = require('lodash')
+
+class Placement {
+  static get QUERY () {
+    return 'SELECT ' +
+      'CAST(to_char(AVG(scores.unaware), \'FM9.999\') AS NUMERIC) as unaware, ' +
+      'CAST(to_char(AVG(scores.curious), \'FM9.999\') AS NUMERIC) as curious, ' +
+      'CAST(to_char(AVG(scores.follower), \'FM9.999\') AS NUMERIC) as follower, ' +
+      'CAST(to_char(AVG(scores.guide), \'FM9.999\') AS NUMERIC) as guide, ' +
+      'CAST(to_char(AVG(scores.unaware * (scores.confidence/100)), \'FM9.999\') AS NUMERIC) as weighted_unaware, ' +
+      'CAST(to_char(AVG(scores.curious * (scores.confidence/100)), \'FM9.999\') AS NUMERIC) as weighted_curious, ' +
+      'CAST(to_char(AVG(scores.follower * (scores.confidence/100)), \'FM9.999\') AS NUMERIC) as weighted_follower, ' +
+      'CAST(to_char(AVG(scores.guide * (scores.confidence/100)), \'FM9.999\') AS NUMERIC) as weighted_guide ' +
+      'FROM events ' +
+      'INNER JOIN scores as scores ON scores.uri = events.uri ' +
+      'WHERE events.user_id = :user_id AND events.created_at > current_timestamp - interval \'90\' day'
```

Comment:
Only looking back 90 days? I thought I remember hearing 1 year or something. Maybe I misremember.

Reply:
I couldn't remember what was decided, so I just used 90 days for now. I could make this an ECS option if needed.

Reply:
I don't think that's necessary, we can just update it if we learn of a different value I guess.